cogs/seasonstats.py

```diff
@@ -521,6 +519,129 @@ def __init__(self, bot):
         p = SeasonStatsPaginator(ctx, entries=entries)
         await p.paginate()
 
+    @season_stats.command(name='attacks')
+    async def season_stats_attacks(self, ctx, season: typing.Optional[int] = None):
+        """Get attack wins for all clans.
+
+           By default, you shouldn't need to call these sub-commands as the bot will
```

Comment:
Pretty sure this isn't true for this case. I'm also unsure as to whether we should document the `season` parameter. I generally haven't been doing that as trying to keep it as ~~dumb~~ simple as possible, but this one might be worth it

Reply:
Removed info on sub-commands. I agree that we probably shouldn't document `+season` since, on its own, it doesn't do anything.